e2e/llm_util_test/test_schema_prompt_ready_with_relationship_props/test.yml

```diff
@@ -0,0 +1,6 @@
+query: >
+    CALL llm_util.schema('prompt_ready') YIELD schema RETURN schema;
+output:
+ - schema: "Node properties are the following:\nNode name: 'Person', Node properties: [{'property': 'name', 'type': 'str'}, {'property': 'age', 'type': 'int'}, {'property': 'year', 'type': 'str'}]\nNode name: 'Student', Node properties: [{'property': 'name', 'type': 'str'}, {'property': 'age', 'type': 'int'}, {'property': 'year', 'type': 'str'}]\nNode name: 'University', Node properties: [{'property': 'name', 'type': 'str'}]\nNode name: 'City', Node properties: [{'property': 'name', 'type': 'str'}]\n\nRelationship properties are the following:\nRelationship Name: 'IS_FRIENDS_WITH', Relationship Properties: [{'property': 'since', 'type': 'str'}]\n\nThe relationships are the following:\n['(:Person)-[:IS_FRIENDS_WITH]->(:Person)']\n['(:Person)-[:IS_FRIENDS_WITH]->(:Student)']\n['(:Person)-[:LIVES_IN]->(:City)']\n['(:Person)-[:STUDIES_AT]->(:University)']\n['(:Student)-[:STUDIES_AT]->(:University)']\n['(:Student)-[:LIVES_IN]->(:City)']\n"
+
+ 
```

Comment:
watch out for these minus signs, you should have everywhere only one line empty after the query (should have 5 lines)